src/hooks/useAuthorityLinking/useAuthorityLinking.js

```diff
@@ -242,6 +251,53 @@ const useAuthorityLinking = () => {
     };
   }, []);
 
+  const actualizeNewLinkedFields = useCallback(async (formValues) => {
+    const isNewLinkedField = (field) => field.linkDetails?.status === AUTOLINKING_STATUSES.NEW;

```

Comment:
We need to update all links, not just new ones to make sure all links are in sync with authority records